.github/workflows/arm64.yml

```diff
@@ -11,13 +11,16 @@ env:
   MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
   TAS_ENVIRONMENT: ./tests/environment
   TAS_SCRIPTS: ../alfresco-community-repo/packaging/tests/scripts
+  JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
   AWS_REGION: eu-west-1
   DTAS_VERSION: v1.1.3
 
 jobs:
   arm64_health_check:
     name: ARM64 Health Check
     runs-on: [self-hosted, linux, arm64]
+    env:
+      BUILD_WEB_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
Could this be an env var of the specific step that needs it, or does it need to be declared at the job-level to work?

Reply:
I believe this can be env var of the specific step. Done.
Ref. https://github.com/Alfresco/acs-packaging/pull/2450